languages/en.ini

```diff
@@ -109,6 +109,7 @@ Availability = "Availability"
 Available = "Available"
 Available Functionality = "Available Functionality"
 Awards = "Awards"
+back_to_browse_by = "Back to Browse %%field%%"
```

Comment:
Looks like this is not quite in the expected location. You can run `php $VUFIND_HOME/public/index language normalize languages` and commit the resulting changes to get the build to pass here.

Reply:
When I run the command it doesn't change the file and the line is at the correct location
The check however seems to detect it at another place and want to move it where it currently is
Could I miss a caching issue or does the pipeline modify the file?

Reply:
I spotted it and corrected it before running it
I realized what was the problem, we have a duplicate of the config files (one in the git repo the other in a local conf symlinked dir) and I copied the wrong one